Makefile

```diff
@@ -188,8 +188,8 @@ $(LOCALBIN):
 
 ## Tool Binaries
 KUSTOMIZE ?= $(LOCALBIN)/kustomize
-CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
-ENVTEST ?= $(LOCALBIN)/setup-envtest
+CONTROLLER_GEN ?= controller-gen
```

Comment:
Why we need to download this binaries in project's root folder?
It will be bettere if there is no other way to do it to add controller-gen and setup-envtest in to the gitignore at least

Reply:
This does not download in root folder.
With `go install`, the binaries will be installed into the GOPATH, which should be the way of doing IMO
we should not need to install in a "local" bin those binaries

Reply:
This part of the Makefile is coming from the operator-sdk (when we initialize a new project), so it is probably better to understand why they are suggesting this approach instead of using the default GOPATH for the go install.

In the meanwhile we could maybe set LOCALBIN=$GOPATH if there isn't a huge effort, what do you think?

Reply:
ok then I will fallback those changes

Reply:
fallback done